_posts/2019-03-06-generating-print-quality-plots.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Generating print-quality plots in R
+author: Danny Eisenberg
+categories: plots, R, RStudio
+---
+Some students have mentioned that they had generated plots with R and submitted them in their theses, but were requested to redo the plots at high resolution and to change the colour palette.
+
+## High Resolution
+
+If you use RStudio, you can export your plots to a file in either PDF or PNG format. If you save it in PDF format, by default it is high resoultion. If, however, you wish to save it as a PNG bitmap, the default resolution will be 72 dpi (dots per inch), the standard screen resolution, far too low for high quality printing, which needs to be at least 300 dpi.
+
+To see what I mean, let's generate a plot as an example. We are will use a sample file recording maximum temperature available from the [UK Met Office](https://www.metoffice.gov.uk/hadobs/hadghcnd/download.html): https://www.metoffice.gov.uk/hadobs/hadghcnd/data/HadGHCND_1949-2011_TXx.nc.gz
```

Comment:
I’m being annoying, but I did wonder if it wouldn’t be better to use a opendap accessible dataset so a user could just paste the code in without having to download and uncompress a data file?